packages/config/package.json

```diff
@@ -40,6 +40,7 @@
     "glob": "7.1.6",
     "globby": "^11.0.0",
     "invariant": "^2.2.4",
+    "jimp": "^0.10.3",
```

Comment:
AFAIK we would like to use our own package https://github.com/expo/expo-cli/tree/master/packages/image-utils for any `sharp`- or `jimp`-based image processing.
We should depend on this package and remove any direct `sharp` or `jimp` dependencies. Even further: if any new better-performing package would appear we would like to support in via `image-utils` package.